src/main/resources/application.yml

```diff
@@ -0,0 +1,27 @@
+spring:
```

Comment:
Il file application.yml non va mai pushato in quanto contiene dati che entrano in conflitto tra i nostri dispositivi, dobbiamo eliminarlo dall'intera repository.

Reply:
Ho appena controllato l'issue di lorenzo ed effettivamente è bene fare come dice lui